src/OpenWeatherMapApi/OpenWeatherMapResources.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenWeatherMapApi
+{
+	public class APIKeys
+	{
+		public struct OpenWeatherMapAPIKey
+		{
+			private string KeyID { get; set; }
```

Comment:
What convention are we sticking to here. Private variable so should probably be `_keyID`